_data/projects/mono-mono.yml

```diff
@@ -0,0 +1,10 @@
+name: Mono
+desc: An open source implementation of Microsoft's .NET Framework based on the ECMA standards for C# and the Common Language Runtime.
+site: https://mono-project.com/
+tags:
+- C#
+- C
+- ".NET"
+upforgrabs:
+  name: good first issue
+  link: https://github.com/mono/mono/labels/good%20first%20issue
```

Comment:
This label doesn't yet exist on their issue tracker

Reply:
What do you mean? The label exists but it's not yet used for any issue

Reply:
Oops, I guess I skimmed over it in the list - apologies!